README.md

```diff
@@ -1,2 +1,24 @@
-# hw01
-Repository for homework 01
+# HW01
+## Deven Mukkamala, Biography
+
+![Deven Mukkamala](https://www.legion.org/sites/legion.org/files/legion/boys-nation-delegates/20170721_Michigan%20Federalist%20Deven%20Mukkamala_7501.jpg)
```

Comment:
Great use of embedding a photo from an external site. If it wasn't hosted on another web page (for instance, something just on your computer), you could put the image in your Git repo and embed it from there.